clickhouse-admin/api/src/lib.rs

```diff
@@ -2,56 +2,38 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-use clickhouse_admin_types::config::{KeeperConfig, ReplicaConfig};
 use clickhouse_admin_types::{
-    ClickhouseKeeperClusterMembership, KeeperConf, KeeperSettings, Lgif,
-    RaftConfig, ServerSettings,
+    ClickhouseKeeperClusterMembership, KeeperConf, KeeperConfig,
+    KeeperConfigurableSettings, Lgif, RaftConfig, ReplicaConfig,
+    ServerConfigurableSettings,
 };
 use dropshot::{
     HttpError, HttpResponseCreated, HttpResponseOk, RequestContext, TypedBody,
 };
-use omicron_common::api::external::Generation;
-use schemars::JsonSchema;
-use serde::{Deserialize, Serialize};
-
-#[derive(Debug, Serialize, Deserialize, JsonSchema)]
-pub struct ServerConfigurableSettings {
-    /// A unique identifier for the configuration generation.
-    pub generation: Generation,
-    /// Configurable settings for a ClickHouse replica server node.
-    pub settings: ServerSettings,
-}
-
-#[derive(Debug, Serialize, Deserialize, JsonSchema)]
-pub struct KeeperConfigurableSettings {
-    /// A unique identifier for the configuration generation.
-    pub generation: Generation,
-    /// Configurable settings for a ClickHouse keeper node.
-    pub settings: KeeperSettings,
-}
 
+/// We separate the admin interface for the keeper and server APIs because they
```

Comment:
First paragraph of doc comments should be short (a lint for this is coming in Rust 1.83.) Could you just add something simple?

Reply:
Thanks for the tip! Done in [0a2e28b](https://github.com/oxidecomputer/omicron/pull/6837/commits/0a2e28b22169bf55587164395a620fb00bef3d4c)